clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookMessageHandler.ts

```diff
@@ -124,3 +147,17 @@ export function downloadCopybookHandler(
     quietMode,
   );
 }
+
+export function resolveCopybookHandler(
+  this: any,
+  documentUri: string,
+  copybookName: string,
+  dialectType: string,
+): string {
+  return this.resolveCopybookHandler(
+    documentUri,
+    copybookName,
+    dialectType,
+    this.storagePath,
+  );
+}
```

Comment:
This seems like a identical situation to the `resolveSubroutineURI`, doesn't it?

Reply:
should resolve : d0998c4b615b9c7d9a81f640527f418178e2ab09